### PR TITLE
Update SCP-029 and SCP-030

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

File: proposals/README.md [hunk 1]
```diff
@@ -164,17 +164,23 @@
 * Status: **completed**
 # [027-refactoring.md](./027-refactoring.md)
 * Date proposed: November 2021
+* Accepted: yes
 * Updates:
 	* revised March 2022
 	* A working group will be created to further pull out actionable items from this proposal before any work will begin.
 	* revised in June 2022 to show the outcome of the working group.
-* Status: **accepted**
+* Status: **dormant**
```
Comment (ckipp01) [inline proposals/README.md:172]: Commented on the other pr about this, but I though we were going to wait on this?

Comment (SethTisue) [reply proposals/README.md:172]: I reverted the other PR, and I marked this one as "draft" until I have a chance to review the meeting video.

Comment (julienrf) [reply proposals/README.md:172]: Yes, we will wait a bit more before merging this change.

Comment (SethTisue) [reply proposals/README.md]: Yeah in the meeting video we said we'd wait.

**Comment by SethTisue** [comment]
I took out the change to SCP-027's status. The discussion at the meeting was a bit inconclusive — there was a mention of waiting a month or so, but it wasn't clear to me if there was a plan to gather further feedback.

In any case, this PR is now ready for merge, afaics?

**Comment by ckipp01** [comment]
> but it wasn't clear to me if there was a plan to gather further feedback.

I'll circle back around with @eed3si9n since it was mentioned that he'd possible gather some more info. I can help here too if needed.

**Review by ckipp01 — APPROVED** [review]
Looks good to me, thanks @SethTisue!

**Pull request merged**